app/Livewire/Server/Proxy.php

```
@@ -144,4 +146,135 @@ public function loadProxyConfiguration()
+                    ->sortBy(fn ($v) => $v, SORT_NATURAL)
+                    ->last();
+
+                return $newestVersion ? "v{$newestVersion}" : null;
+            }
+
+            // Handle old structure (simple string) for backward compatibility
+            return str_starts_with($traefikVersions, 'v') ? $traefikVersions : "v{$traefikVersions}";
+        } catch (\Throwable $e) {
+            return null;
+        }
+    }
```

PHPMD (2.15.0)

The method getLatestTraefikVersionProperty() has a Cyclomatic Complexity of 12. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)